.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+Thank you for your pull request. Please provide the following information.
+
+---
+
+> **Do changes in this PR impact **User Experience** (e.g., how the user interacts with Yo Office and/or the files and folders the user sees in the project that Yo Office creates)?**
```

Comment:
I would prefer we not use the abbreviation "PR". @kbrandl Can the wording be changed to "Do these changes impact..." ?